qlib/contrib/model/pytorch_alstm_ts.py

```diff
@@ -321,7 +320,7 @@ def _build_model(self):
         try:
             klass = getattr(nn, self.rnn_type.upper())
         except:
-            raise ValueError("unknown rnn_type `%s`" % self.rnn_type)
+            raise ValueError("unknown rnn_type `%s`" % self.rnn_type) from None
```

Comment:
Similar to the above error 
![image](https://user-images.githubusercontent.com/465606/151107204-3c13361f-1c75-4e08-b1bd-87efd1025ae7.png)